packages/accordion/README.md

```diff
@@ -1,13 +1,10 @@
-# &lt;vaadin-accordion&gt;
+# @vaadin/accordion
 
-[&lt;vaadin-accordion&gt;](https://vaadin.com/components/vaadin-accordion) is a Web Component implementing the vertically stacked set of expandable panels, part of the [Vaadin components](https://vaadin.com/components).
+A web component implementing accordion widget: a vertically stacked set of expandable panels.
 
-[Live Demo ↗](https://vaadin.com/components/vaadin-accordion/html-examples)
-|
-[API documentation ↗](https://vaadin.com/components/vaadin-accordion/html-api)
+[Live Demo ↗](https://vaadin.com/docs/latest/ds/components/accordion)
```

Comment:
```suggestion
[Documentation + Live Demo ↗](https://vaadin.com/docs/latest/ds/components/accordion)
```
We should still mention the documentation, for users it's not obvious that the live demo will actually point to the docs.

Reply:
Updated. Will do the same for all the `vaadin-` packages in a separate PR.